robusta_krr/core/integrations/prometheus/loader.py

```diff
@@ -17,6 +15,8 @@
 from robusta_krr.utils.configurable import Configurable
 from robusta_krr.utils.service_discovery import ServiceDiscovery
 
+from .metrics import BaseMetricLoader
```

Comment:
fix relative import
`from robusta_krr.core.integrations.prometheus.metrics import BaseMetricLoader`
not sure how critical this is since I see you do it in a bunch of places, you know convention better than me

Reply:
We were replacing relative imports in robusta as we had `from .......src.core import something` 💀 
Relative imports from `.` or `..` can be fine, if those imports are about the linked module, like here: `BaseMetricLoader` is related to all other loaders